schemas/mobility.schema

```diff
@@ -22,18 +22,21 @@ n_trips_250_500 	   INTEGER,
 n_trips_gte_500 	   INTEGER
 );
 
-CREATE TABLE $PG_SCHEMA_MOBILITY.airtraffic (
+CREATE UNLOGGED TABLE $PG_SCHEMA_MOBILITY.airtraffic (
+id 				       SERIAL PRIMARY KEY,
 ts  				   TIMESTAMP NOT NULL,
 origin_code   		   TEXT,
 origin_city   		   TEXT,
 origin_admin0		   TEXT,
 origin_admin1		   TEXT,
 origin_fips			   TEXT,
+origin_locale		   INTEGER REFERENCES $PG_SCHEMA_MAIN.locale (id) ON DELETE CASCADE ON UPDATE CASCADE,
```

Comment:
Almost missed this, but any thoughts on using `origin_locale_id` here for consistency with the `dis` schema?

Reply:
Good call. Done!